distribution/docker/README.md

```diff
@@ -19,22 +19,29 @@
 
 ## Build
 
-From the root of the repo, run `docker build -t apache/druid:tag -f distribution/docker/Dockerfile .`
+From the root of the repo, run following command:
 
-## Run
-
-Edit `environment` to suite. Run `docker-compose -f distribution/docker/docker-compose.yml up`
+```bash
+DOCKER_BUILDKIT=1 docker build -t apache/druid:tag -f distribution/docker/Dockerfile .
```

Comment:
I was wondering if the need for buildkit would impact our Docker Hub autobuilds, but it looks like it's already enable by default on docker hub https://docs.docker.com/docker-hub/builds/#build-images-with-buildkit
(we still need it in the docs of course)

Reply:
do we need to enable buildkit when we build our integration test images?

Reply:
Our it test images are not based on the distribution docker file, so I don't touch this part in this PR. 
We can enable it and wait to see if the travis works fine well with buildkit in a follow up PR.

Reply:
we do for our k8s integration tests, where we use the default image.https://github.com/apache/druid/blob/master/integration-tests/script/setup_druid_on_k8s.sh#L35-L36

Reply:
Let me update the command in this file